### PR TITLE
SEO-177279-Page-With-Redirect-React1

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on links with redirects.

Regards,
Beryl

### CODE REVIEW

File: ej2-react/licensing/license-key-registration.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ The following steps show how to register the Syncfusion license key with the lic
 
 * Remove the `.cache` folder from node modules in the application.
 
-* Now run the application. If you are facing a license validation error, refer to this [link](https://ej2.syncfusion.com/react/documentation/licensing/licensing-errors/) to resolve it. Also, find the most frequent license registration questions from this [`link`](https://ej2.syncfusion.com/react/documentation/licensing/licensing-troubleshoot/).
+* Now run the application. If you are facing a license validation error, refer to this [link](hhttps://ej2.syncfusion.com/react/documentation/licensing/licensing-errors) to resolve it. Also, find the most frequent license registration questions from this [`link`](https://ej2.syncfusion.com/react/documentation/licensing/licensing-troubleshoot).
```
Comment (Yvone-Atieno) [inline ej2-react/licensing/license-key-registration.md:74]: Check the first link

File: ej2-react/grid/filtering/filter-bar.md [hunk 2]
```diff
@@ -76,4 +76,4 @@ In the following sample dropdown is used  as custom component in **EmployeeID**
 
  ## See also
 
-* [How to perform filter by using Wildcard and LIKE operator filter](./filtering/#wildcard-and-like-operator-filter)
\ No newline at end of file
+* [How to perform filter by using Wildcard and LIKE operator filter](https://ej2.syncfusion.com/react/documentation/grid/filtering/filtering#wildcard-and-like-operator-filter)
```
Comment (Yvone-Atieno) [inline ej2-react/grid/filtering/filter-bar.md:79]: Revert the https link, and use as it was previously

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: ej2-react/licensing/license-key-registration.md (hunk 1 above)
Comment (berylouma) [reply ej2-react/licensing/license-key-registration.md]: Corrected

File: ej2-react/grid/filtering/filter-bar.md (hunk 2 above)
Comment (berylouma) [reply ej2-react/grid/filtering/filter-bar.md]: Reverted

**Pull request closed without merging**